src/app/component/auth/sign-in-new/sign-in-new.component.ts

```diff
@@ -19,20 +21,91 @@ export class SignInNewComponent implements OnInit {
   private googleImage = SignInIcons;
   private hideShowPasswordImage = SignInIcons;
 
+  public userOwnSignIn: UserOwnSignIn;
+  public loadingAnim: boolean;
+  public emailErrorMessageBackEnd: string;
+  public passwordErrorMessageBackEnd: string;
+  public backEndError: string;
+
   constructor(
     private matDialogRef: MatDialogRef<SignInNewComponent>,
     private userOwnSignInService: UserOwnSignInService,
     private router: Router,
     private authService: AuthService,
-    private googleService: GoogleSignInService
+    private googleService: GoogleSignInService,
+    private localStorageService: LocalStorageService,
   ) { }
 
-  ngOnInit() {}
+  ngOnInit() {
+    this.userOwnSignIn = new UserOwnSignIn();
```

Comment:
Please divide this code into several methods and add subscription

Reply:
Fixed